packages/sdk/schema/src/view.ts

```diff
@@ -76,6 +78,33 @@ export class ViewType extends TypedObject({
   // TODO(burdon): Add array of sort orders (which might be tuples).
 }) {}
 
+// TODO(wittjosiah): Refactor to organize better previous versions + migrations.
+export class ViewTypeV1 extends TypedObject({
+  typename: 'dxos.org/type/View',
+  version: '0.1.0',
+})({
+  name: S.String.annotations({
+    [AST.TitleAnnotationId]: 'Name',
+    [AST.ExamplesAnnotationId]: ['Contact'],
+  }),
+  query: S.Struct({
+    type: S.optional(S.String),
+    sort: S.optional(S.Array(FieldSortType)),
+  }).pipe(S.mutable),
+  schema: S.optional(JsonSchemaType),
+  fields: S.mutable(S.Array(FieldSchema)),
+  metadata: S.optional(S.Record({ key: S.String, value: S.Any }).pipe(S.mutable)),
```

Comment:
not a huge issue but lets think of a pattern on how to avoid repetiion when versioning schema. e.g. using S.Struct(...).fields and spreading them

Reply:
agreed